### ISSUE TITLE
Support for CosmosDB backup policy

### ISSUE THREAD

**Issue by apielasa** [terraform-providers/terraform-provider-azurerm#8507]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I'd like to set a custom backup policy for CosmosDB. Default values of intervals and retention are not sufficient for production setup.

### New or Affected Resource(s)

* azurerm_2.26.0

### Potential Terraform Configuration

It would be beneficial to set this kind of parameters:

```hcl
  backup {
    frequency = "Daily"
    time      = "23:00"
  }

  retention_daily {
    count = 30
  }
```

**Comment by yupwei68** [comment]
Hi @apielasa , thanks for opening this issue. Since the service has not supported the backup feature, would you like to open an issue [here](https://github.com/Azure/azure-rest-api-specs/issues) to suggest the service team for this feature?

**Comment by tombuildsstuff** [comment]
Support for this is available in the Preview API version: https://github.com/Azure/azure-sdk-for-go/blob/69278854b6aaf085e0a564ee63a0a6d61e2ad446/services/preview/cosmos-db/mgmt/2020-06-01-preview/documentdb/models.go#L139-L160

**Comment by apielasa** [comment]
Hi, @yupwei68 & @tombuildsstuff, Thank you for responding to this issue. I guess I should not open a new feature request if it's already in preview as @tombuildsstuff  mentioned.
Do you have any idea when this feature will be released?

**Comment by apielasa** [comment]
It looks like this feature is has been released with release v45.0.0 of azure-sdk-for-go

**Comment by bugblatter2016** [comment]
Hi, any idea when this will looked at?
